components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -47,13 +47,16 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
     });
 
     // the addPolicies checkbox should always be checked by default when the modal opens
-    this.resetCheckboxEvent.pipe(takeUntil(this.isDestroyed))
-      .subscribe(() => {
-        // the checkbox needs to use this component variable
-        // but we must ensure the createForm.addPolicies value stays in sync
-        this.addPolicies = true;
-        this.createForm.controls.addPolicies.setValue(this.addPolicies);
-      });
+    if (this.resetPoliciesCheckboxEvent) {
+      this.createProjectModal = true;
+      this.resetPoliciesCheckboxEvent.pipe(takeUntil(this.isDestroyed))
+        .subscribe(() => {
+          // the checkbox needs to use this component variable
+          // but we must ensure the createForm.addPolicies value stays in sync
+          this.addPolicies = true;
+          this.createForm.controls.addPolicies.setValue(this.addPolicies);
```

Comment:
```suggestion
          this.createForm.controls.addPolicies.setValue(true);
```

I don't know? What do you think?

Reply:
Your question got me thinking... and realized there was some simplification at hand. Thanks!

Reply:
i like this suggestion